README.md

```diff
@@ -118,6 +118,65 @@ Whilst the `asset-pipline` script is custom, it leverages a lot of existing libs
 
 For the `asset-pipeline` script to do its thing, all you need to do is refer to all assets with a path beginning with `/blog/assets/`. If you do that, everything else is handled for you ✨
 
+## Deployment strategy
+
+Currently, we use a "simple" deployment strategy based on git tags and Circle CI to deploy this blog. We have three different tag patterns:
+
+- `<year>.<month>.<date>-stage` (e.g., `2021.07.22-stage`): these tags will be deployed to our [-stage instance][stage]
+- `<year>.<month>.<date>` (e.g., `2021.07.22`): these tags will be deployed to our [-prod instance][prod]
+- `x.y.z` (e.g., `1.3.0`): this is the `version` in the `package.json` and it is used to version the WordPress theme that can be constructed in this project
+
+Each deployment will fetch the content from the (headless) WordPress instance, which ensures that the most recent content will be deployed.
```

Comment:
It might be good to talk about how publishing works in terms of wordpress. E.g. publishing the blog through a tag will only pull the content available to it at that point in time. If content is embargoed (publish date in the future) on WP it will not be visible to the build process.

Reply:
I updated this paragraph a bit. I am wondering what we're doing with the content published in the future, though. I think it isn't exposed in the WP API response but I don't remember.. Anyway, Editorial makes the content private by default and only public when it's ready.